src/api/app/assets/javascripts/webui2/project_monitor.js

```diff
@@ -61,7 +66,14 @@ function initializeMonitorDataTable() {
         data: null,
         className: 'text-center',
         render: function (packageName, type, row, meta) {
-          return statusCell(meta, statusHash, tableInfo, projectName, packageName);
+          var cellContent = statusCell(meta, statusHash, tableInfo, projectName, packageName);
+          if (cellContent === null) return null;
+          // Value to display in the cell
+          if (type === 'display') {
```

Comment:
Nitpick: this `if` can also be a onliner, like the one two lines above.

Reply:
done!